src/sequences/SequenceEntity.js

```diff
@@ -18,19 +18,32 @@ export class SequenceEntity extends Entity {
 		return this._entity && this._entity.properties && this._entity.properties.title;
 	}
 
+	sequenceViewerApplicationHref() {
+		return this._entity && this._entity.hasLinkByRel('https://sequences.api.brightspace.com/rels/sequence-viewer-application') &&
```

Comment:
Can we use constant for these links? At least, I think the hostname should be a constant.